index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare function getCountryISO2(iso3: string): string;
```

Comment:
I know that all countries are already there and defined, but wouldn't it be more correct if the return value was `string | undefined`?

Reply:
This makes sense, when will this be merged?